### PR TITLE
Simplify algorithm prose for starting, joining, establishing connection; use common idioms

### PR DESCRIPTION
**Pull request by anssiko**
Closes w3c/presentation-api#65.

### CODE REVIEW

File: Overview.src.html [hunk 1]
```diff
@@ -6,14 +6,14 @@
       Presentation API
     </title>
     <style>
-    @media print {
+@media print {
```
Comment (mfoltzgoogle) [inline Overview.src.html]: Somehow this line keeps getting indented and un-indented by consecutive PRs.  Is there some difference in our version of tidy-html5?

File: Overview.src.html [hunk 2]
```diff
@@ -6,14 +6,14 @@
...
       [data-anolis-spec]::after { content:"[" attr(data-anolis-spec) "]"; font-size:.6em; vertical-align:super; text-transform:uppercase }
     }
     </style>
     <link rel="stylesheet" type="text/css" href=
     "http://www.w3.org/StyleSheets/TR/W3C-[STATUS]">
     <style type="text/css">
-    /* Note formatting taken from HTML5 spec */
+/* Note formatting taken from HTML5 spec */
```
Comment (mfoltzgoogle) [inline Overview.src.html]: This line too.

**Comment by mfoltzgoogle** [comment]
LGTM

File: Overview.src.html (hunk 1 above)
Comment (anssiko) [inline Overview.src.html]: I upgraded to HTML Tidy for Mac OS X version 4.9.35 (the latest tagged release). That seem to have fixed the issue.

File: Overview.src.html (hunk 2 above)
Comment (anssiko) [inline Overview.src.html]: As well as this one.

**Pull request merged**